docs/lib/restapi/fragments/js/authz.md

```diff
@@ -13,13 +13,28 @@ Amplify.configure({
           return { Authorization : 'token' } 
           // Alternatively, with Cognito User Pools use this:
           // return { Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}` }
+          // return { Authorization: `Bearer ${(await Auth.currentSession()).getIdToken().getJwtToken()}` }
         }
       }
     ]
   }
 });
 ```
 
+## Note related to use Access Token or ID Token
+
+The ID Token contains claims about the identity of the authenticated user such as name, email, and phone_number. It could have custom claims as well (For example using [Amplify CLI](https://docs.amplify.aws/cli/usage/lambda-triggers#override-id-token-claims) ). On Amplify Auth category you can retrieve the Id Token using: 
+
+```javascript
+(await Auth.currentSession()).getIdToken().getJwtToken()
+``` 
+
+The Access Token contains scopes and groups and is used to grant access to authorized resources. [This is a tutorial for enabling custom scopes.](https://aws.amazon.com/premiumsupport/knowledge-center/cognito-custom-scopes-api-gateway/). You can retrieve the Access Token using 
+
+```javascript
+(await Auth.currentSession()).getAccessToken().getJwtToken()
```

Comment:
```suggestion
await Auth.currentSession()).getAccessToken().getJwtToken();
```

Reply:
`(await Auth.currentSession())` that is need to get the token